### PR TITLE
Update the Wish List Search page information and describe the difference between private and public wishlist

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request:
- Updates the Wish List Search page information 
- Describes the difference between private and public wishlist

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- yes

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/wishlist-search.html
https://docs.magento.com/user-guide/customers/account-dashboard-my-wish-list.html

whatsnew
Updated the [Wish List Search](https://docs.magento.com/user-guide/marketing/wishlist-search.html) topic to include information about searching for another customer's public wish list on the storefront.

### CODE REVIEW

File: src/marketing/wishlist-search.md [hunk 1]
```diff
@@ -77,3 +77,22 @@ _Wish list search_
 1. When complete, click <span class="btn">Save</span>.
 
 1. When prompted, refresh all invalid caches.
+
+## How to find a customer's wishlist
```
Comment (leslietilling) [inline src/marketing/wishlist-search.md:81]:
```suggestion
## Find a customer's wishlist
```

File: src/customers/account-dashboard-my-wish-list.md [hunk 2]
```diff
@@ -22,6 +22,9 @@ In the storefront, a customer can create a wish list from their account dashboar
 
 1. To allow others to see their wish list, selects the **Public Wish List** checkbox.
 
+    {:.bs-callout-info}
+    The main difference between `Public` and `Private` wishlists is that private wishlists are not searchable through [wishlists]({% link marketing/wishlist-search.md %}#how-to-find-a-customers-wishlist).
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-wish-list.md:26]:
```suggestion
    The main difference between `Public` and `Private` wishlists is that private wishlists are not searchable through [wishlists]({% link marketing/wishlist-search.md %}#find-a-customers-wishlist).
```

File: src/marketing/wishlist-search.md [hunk 3]
```diff
@@ -77,3 +77,22 @@ _Wish list search_
...
+
+1. From the Wish List Search widget select a search option.
```
Comment (leslietilling) [inline src/marketing/wishlist-search.md:83]:
```suggestion
1. From the Wish List Search widget, select a search option.
```

File: src/marketing/wishlist-search.md [hunk 4]
```diff
@@ -77,3 +77,22 @@ _Wish list search_
...
+
+1. Enter the customer's name or email address and click <span class="btn">Search</span>.
+
+   The **Wish List Search** page will opens, found wishlists are displayed in the search results section.
```
Comment (leslietilling) [inline src/marketing/wishlist-search.md:87]:
```suggestion
   The **Wish List Search** page opens and any matching wishlists are displayed in the search results section.
```

File: src/marketing/wishlist-search.md [hunk 5]
```diff
@@ -77,3 +77,22 @@ _Wish list search_
...
+
+   {:.bs-callout-info}
+   Only public wishlists can be displayed in search results, customer`s privat wishlists are not publicly viewable.
```
Comment (leslietilling) [inline src/marketing/wishlist-search.md:90]:
```suggestion
   Only public wishlists are displayed in search results---customers' private wishlists are not publicly viewable.
```

File: src/marketing/wishlist-search.md [hunk 6]
```diff
@@ -77,3 +77,22 @@ _Wish list search_
...
+
+1. To view the list of customer wishes, click <span class="btn">View</span>.
+
+   For each customer's wishlist displays the owner name and the date of the last wishlist update.
```
Comment (leslietilling) [inline src/marketing/wishlist-search.md:94]:
```suggestion
   The owner name and the date of the last update is  displayed for each wishlist.
```

File: src/marketing/wishlist-search.md [hunk 7]
```diff
@@ -77,3 +77,22 @@ _Wish list search_
...
+
+1. To add a product to your cart, check the box below the product and click <span class="btn">Add to Cart</span>.
```
Comment (leslietilling) [inline src/marketing/wishlist-search.md:96]:
```suggestion
1. To add a product to your cart, select the checkbox below the product and click <span class="btn">Add to Cart</span>.
```

File: src/marketing/wishlist-search.md [hunk 8]
```diff
@@ -77,3 +77,22 @@ _Wish list search_
...
+
+   In addition, you can add items you like from another customer's wishlist to your own.
```
Comment (leslietilling) [inline src/marketing/wishlist-search.md:98]:
```suggestion
   You can also add items you like from another customer's wishlist to your own.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Minor edits for style and clarity

**Comment by ZhovnirMaria** [comment]
@leslietilling I have updated my PR, please check) Thanks ☺️

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**